src/preprocessing/passes/bv_to_int.cpp

```diff
@@ -440,16 +440,38 @@ Node BVToInt::translateWithChildren(Node original,
         // translate the result to integers
         returnNode = d_nm->mkNode(kind::BITVECTOR_TO_NAT, bvand);
       }
-      else
+      else if (options::solveBVAsInt() == options::SolveBVAsIntMode::SUM)
       {
-        Assert(options::solveBVAsInt() == options::SolveBVAsIntMode::SUM);
         // Construct a sum of ites, based on granularity.
         Assert(translated_children.size() == 2);
         returnNode =
-            d_iandTable.createBitwiseNode(translated_children[0],
-                                          translated_children[1],
-                                          bvsize,
-                                          options::BVAndIntegerGranularity());
+            d_iandUtils.createSumNode(translated_children[0],
+                                      translated_children[1],
+                                      bvsize,
+                                      options::BVAndIntegerGranularity());
+      }
+      else
+      {
+        Assert(options::solveBVAsInt() == options::SolveBVAsIntMode::BITWISE);
+        uint64_t granularity = options::BVAndIntegerGranularity();
+
+        Node x = translated_children[0];
+        Node y = translated_children[1];
+        returnNode = d_nm->mkNode(kind::IAND, x, y);
```

Comment:
`kind::IAND` is parametric, so u need to first create an operator and then use it in the node:
https://github.com/CVC4/CVC4/blob/a6c001e078767a2de6f36a2fa1333b98e39a6ec8/src/preprocessing/passes/bv_to_int.cpp#L426

Reply:
Oops, thanks for catching that! Would it be hard to add a unit test for this pass? Maybe I should do that.

Reply:
Fixed with 9720d2985d30aa3f864e0f3e8d5107f7bcdc417b